torch/distributed/distributed_c10d.py

```diff
@@ -4239,6 +4264,9 @@ def _get_group_tag(pg: ProcessGroup) -> str:
 def _get_process_group_name(pg: ProcessGroup) -> str:
     return _world.pg_names.get(pg, "None")
 
+def _get_process_group_desc(pg: ProcessGroup) -> str:
+    return pg.group_desc
+
```

Comment:
nit: is this function used anywhere? Does it have some functionality as the `group_desc` property?